### DISCUSSION TITLE
Lesson 3: Fund Me

### DISCUSSION THREAD

**Discussion by propeltrain** [smartcontractkit/full-blockchain-solidity-course-py#1808] [Q&A] [answered]
What is the meaning and use of  +=   between [msg.sender] and msg.value?

Timestamp: 2.29.46 minutes. 

![02323B22-3825-4158-B084-A8E8FDFF3474](https://user-images.githubusercontent.com/109322953/206945170-37fa57b5-4023-4f9f-a671-0ac4797a0d90.jpeg)

**Comment by Niferu** [comment, marked as answer]
Hello @propeltrain 

y += x is equal to y = y + x

so in your case it will be:

```solidity
addressToAmountFunded[msg.sender] += msg.value

//will be equal to

addressToAmountFunded[msg.sender] = addressToAmountFunded[msg.sender] + msg.value
```
Let me know if it is clear enough for you :)

**Reply by cromewar** [reply]
Nice answer @Neftyr!!

**Reply by propeltrain** [reply]
Thank you so much :). It’s clear now. @Neftyr @cromewar
